src/Sylius/Bundle/AdminBundle/DependencyInjection/SyliusAdminExtension.php

```diff
@@ -33,6 +33,13 @@ public function load(array $config, ContainerBuilder $container): void
 
         $container->setParameter('sylius.admin.notification.enabled', $config['notifications']['enabled']);
         $container->setParameter('sylius.admin.notification.frequency', $config['notifications']['frequency']);
+        $container->setParameter('sylius.admin.shop_enabled', false);
+
+        $bundles = $container->getParameter('kernel.bundles');
+        if (array_key_exists('SyliusShopBundle', $bundles)) {
+            $loader->load('services/shop.xml');
```

Comment:
Let's follow the current pattern of having optional services in `integrations` directory, like `integrations/shop.xml` here.